extensions/amp-bind/validator-amp-bind.protoascii

```diff
@@ -78,3 +78,20 @@ tags: {  # <amp-state>
   }
   spec_url: "https://www.ampproject.org/docs/reference/components/amp-bind"
 }
+tags: {  # <amp-bind-macro>
+  html_format: AMP
+  tag_name: "AMP-BIND-MACRO"
+  requires_extension: "amp-bind"
+  attrs: {
```

Comment:
These still need to be alphabetized.

Reply:
Done. Sorry, I must not have saved this change when I made it earlier.